lib/cisco_node_utils/command_reference.rb

```diff
@@ -339,12 +339,34 @@ def debug(text)
       puts "DEBUG: #{text}" if @@debug
     end
 
+    KNOWN_PLATFORMS = %w(C3064 N3k N5k N6k N7k N9k XRv9k)
+
+    def self.platform_to_filter(platform)
+      case platform
+      when 'C3064'
+        /C3064/
+      when 'N3k'
+        /N3/
+      when 'N5k'
+        /N5/
+      when 'N6k'
+        /N6/
+      when 'N7k'
+        /N7/
+      when 'N9k'
+        /N9/
+      when 'XRv9k'
```

Comment:
This is just mapping the key in the YAML to the corresponding regexp. I didn't have any examples of NX-OSv specific keys in the YAML so I didn't add it yet. We can add later if needed.